account/accounts.go

```diff
@@ -37,8 +35,7 @@ var zeroAddress = types.Address{}
 // Manager represents account manager interface.
 type Manager struct {
```

Comment:
Do we need to keep it exported?

Reply:
For the time being yes, since it's being used in other packages like `multiaccounts` or `accounts`. If the opportunity arises we should have an interface and make the implementation private.